benchmark_results_data/scout_bignum_benchmarks.csv

```diff
@@ -1,4 +1,14 @@
 engine,bench_name,parse_time,exec_time
+wasm3-with-bignums,bls12-wasmsnark-synth-loop,198.70827066171864,5151.1405701083795

```

Comment:
Right now these numbers are not really useful if the fizzy numbers are not run at the same time.

Reply:
Ah good point.  we had a Fizzy version update that isn't accounted for dataset atm.